SPECS/influx-cli/influx-cli.spec

```diff
@@ -17,7 +17,7 @@
 
 Summary:        CLI for managing resources in InfluxDB
 Name:           influx-cli
-Version:        2.4.0
+Version:        2.6.1
```

Comment:
No changelog?

Reply:
I was not sure if I need a changelog for this. This package has not been released yet or built in main.

Reply:
I would keep changelog for every change we did. Please update the specs with an appropriate changelog entry.

Reply:
Updated